anchor/client/src/lib.rs

```diff
@@ -322,25 +324,26 @@ impl Client {
         // Wait until genesis has occurred.
         wait_for_genesis(&beacon_nodes, genesis_time).await?;
 
+        // Start validator index syncer
+        let index_sync_tx =
```

Comment:
It's unrelated to this PR, but could we split the `run` function into smaller ones? It's almost 500 lines long, way too big, IMO.